src/app/shared/delayed-retry/delayed-retry.ts

```diff
@@ -1,12 +1,12 @@
 import { Observable, of, throwError } from 'rxjs';
 import { delay, mergeMap, retryWhen, switchMap } from 'rxjs/operators';
 
-const getErrorMessage = { error: `Unable to load data. Please contact the help desk or refresh the page to try again.`};
+export const getErrorMessage = { error: `Unable to load data. Please contact the help desk or refresh the page to try again.` };
```

Comment:
The export is never used

Reply:
ideally should be in enum if needs to be used in more than one place.

Reply:
this is a specific error only thrown by this method. removing the export